cmd/version/version.go

```diff
@@ -1,26 +1,24 @@
 package version
 
 import (
-	"fmt"
+	"github.com/edgexfoundry-holding/edgex-cli/pkg/cmd/version"
 
-	client "github.com/edgexfoundry-holding/edgex-cli/pkg"
 	"github.com/spf13/cobra"
 )
 
-var version string
-
 // NewCommand returns the version command
 func NewCommand() *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "version",
 		Short: "Version command",
 		Long:  `Outputs the current version of edgex-cli.`,
-		Run: func(cmd *cobra.Command, args []string) {
-			// Printing the CLI's version
-			fmt.Println("edgex-cli version: ", version)
-			// printing the version of each service
-			data := client.GetVersion("48080")
-			fmt.Println("version: ", string(data))
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			version, err := version.GetVersion("48080")
```

Comment:
Earlier it printed out two versions, the edgex-cli version and the edgex-go version.
Now only edgex-go version. Need also the contents of VERSION file in the edgex-cli repo. Unclear to me how that got fetched earlier just looking at the code.

Reply:
Really .. let me check what I have wiped it out